### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by alexander-myltsev**

### CODE REVIEW

File: universal/data/nyse_n.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W
+A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,x1,R,S,T,U,V,W
```
Comment (Marigold) [inline universal/data/nyse_n.csv:1]: Is there a reason for this change?

File: universal/data/nyse_o.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W,X,Y,Z,[,\,],^,_,`,a,b,c,d
+A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,x7,R,S,T,U,V,W,X,Y,Z,x1,x2,x3,x4,x5,x6,a,b,c,d
```
Comment (Marigold) [inline universal/data/nyse_o.csv:1]: Same here, is this just for debugging or an actual change?

**Review by Marigold — APPROVED** [review]

**Comment by Marigold** [comment]
Looks good! Left just one minor comment.

On a side note, if you're experimenting with Kelly portfolio you should try out MPT. It's a generalization of Kelly, is more robust and can be better controlled (e.g. long only etc.).

File: universal/data/nyse_n.csv (hunk 1 above)
Comment (alexander-myltsev) [reply universal/data/nyse_n.csv:1]: There is `OLS.from_formula` fails if there is a `Q` column in a dataset. You're welcome if you have an idea how to solve it better :)

File: universal/data/nyse_o.csv (hunk 2 above)
Comment (alexander-myltsev) [reply universal/data/nyse_o.csv:1]: Same here.

File: universal/data/nyse_n.csv (hunk 1 above)
Comment (Marigold) [reply universal/data/nyse_n.csv:1]: Ohh right, `Q` is a reserved keyword in patsy... I like this hack, don't think there's an easy way around it.

**Comment by alexander-myltsev** [comment]
@Marigold would you merge it?

**Comment by Marigold** [comment]
@alexander-myltsev merged, btw could you add links to relevant papers to docstrings (in some future PR when convenient)? Never heard of these algos and would love to learn more.

**Comment by alexander-myltsev** [comment]
@Marigold I am so sorry :) I messed up git commits. I completely missed that `fixes` branch is at PR. Those both Kelly algorithms implementations are my state-of-the-art implementation and under active research. They simply don't work as expected :) You should revert the PR from me and merge the current `fixes` branch. Or could you grant me temporary access to the repo so I can fix it myself?

**Comment by Marigold** [comment]
No worries, happened to me hundreds of times... Added you as a collaborator, ping me if this doesn't give you sufficient permissions.

**Comment by alexander-myltsev** [comment]
I need rights to `push force` to rewrite history. Could you add the permission?

**Comment by Marigold** [comment]
Enabled force push to the repo. Try it out, if that doesn't work for you I'll do it myself (couldn't find specific collaborator permissions).

**Comment by alexander-myltsev** [comment]
Still doesn't have rights. The least I can do is to give you bash commands:
```bash
git stash
git reset --hard 97726b215a0d
git merge --no-ff 2681ec95a942
git push -f origin master
```

**Comment by alexander-myltsev** [comment]
Thanks. I almost done:

```
g push -f marigold master
Enumerating objects: 1, done.
Counting objects: 100% (1/1), done.
Writing objects: 100% (1/1), 240 bytes | 48.00 KiB/s, done.
Total 1 (delta 0), reused 0 (delta 0)
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: At least 1 approving review is required by reviewers with write access.
To github.com:Marigold/universal-portfolios.git
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:Marigold/universal-portfolios.git'
```

I suggest you turn off those policies until I force push to `master`.

**Comment by alexander-myltsev** [comment]
@Marigold or just force `master` branch to [marigold-master](https://github.com/Marigold/universal-portfolios/tree/marigold-master). And make `push -f` on your side.

**Comment by Marigold** [comment]
Forced pushed myself. Sorry, I should have done it right away. Thanks for help on this.

**Comment by alexander-myltsev** [comment]
Thank you! Again, sorry for the mess.

**Pull request merged**